backend/db/sqliteschema/migrations.go

```diff
@@ -219,4 +219,66 @@ var migrations = []string{
 		-- The balance in satoshis
 		balance INTEGER DEFAULT 0
 	);`,
+
+	// Stores sites that user has manually added
+	`CREATE TABLE sites (
+		-- Site unique identification. The hostname of the site with protocol https://example.com
+		hostname TEXT PRIMARY KEY,
+		-- The role we play in the site ROLE_UNSPECIFIED = 0 | OWNER = 1 | EDITOR = 2
+		role INT NOT NULL DEFAULT 0,
+		-- Address of the LND node backing up this wallet. In case lndhub, this will be the 
+		-- P2P addresses to connect to that site in the format of multiaddresses. Space separated.
+		addresses TEXT NOT NULL,
+		-- The account ID of the site. We need a previous connection to the site so the 
+		-- actual account is inserted in the accounts table when handshake.
+		account_id INTEGER REFERENCES accounts ON DELETE CASCADE NOT NULL
+	) WITHOUT ROWID;`,
+
+	// Table that stores all the tokens not yet redeemed inside a site. Although this table is relevant only
+	// for sites at the beginning, keep in mind that any regular node can be upgraded to a site.
+	`CREATE TABLE unredeemed_invite_links (
```

Comment:
Maybe call them simply invite_tokens or something? They are definitely not links here, I guess, only tokens right?

Reply:
Sure